src/components/Cases/style.scss

```diff
@@ -1,36 +1,36 @@
-#banner-wrapper {
-  // background-color: grey;
-  display: flex;
-  flex-direction: column;
-}
+// #banner-wrapper {
+//   // background-color: grey;
+//   display: flex;
```

Comment:
I think if this is just a commented out file now it's probably best to remove it entirely. Don't need to keep it around.

Reply:
will do!

Reply:
Rest looks great tho :+1:

Reply:
I realise that the Case page still needs this .scss until we have changed those files to styled components!! so I have uncommented it so it renders properly for now!

Reply:
cool will remove it, some styles won't render in the Revision flow but Eve is working on that so they will work once we merge her work